cheerio-scraper/src/crawler_setup.js

```diff
@@ -261,11 +261,6 @@ class CrawlerSetup {
             },
             pageFunctionArguments: {
                 ...crawlingContext,
```

Comment:
This destructuring will actually call the getters on the `crawlingContext`.

Reply:
Should be fixed (the same snippet was in the puppeteer scraper, so addressed it there too). Please take a look if it's correct.